example/lib/main.dart

```diff
@@ -22,6 +22,9 @@ class MyApp extends StatelessWidget {
           checkFontOverflows: true,
           checkImageLabels: true,
           buttonsAlignment: ButtonsAlignment.bottomRight,
+          testEnvironment: const TestEnvironment(
+            visualDensity: VisualDensity.standard,
+          ),
```

Comment:
Could you please remove this change? This file is also used as [pub.dev's example code](https://pub.dev/packages/accessibility_tools/example) so we're trying to keep it simple. Custom test environment is a feature for advanced users so we don't need it to show it there